src/components/accession2/viabilityTesting/ViewViabilityTestModal.tsx

```diff
@@ -0,0 +1,139 @@
+import { Grid, Typography } from '@mui/material';
+import { Button, DialogBox } from '@terraware/web-components';
+import { Accession2 } from 'src/api/accessions2/accession';
+import strings from 'src/strings';
+import { ViabilityTest } from 'src/api/types/accessions';
+import { getFullTestType } from 'src/utils/viabilityTest';
+
+export interface ViewViabilityTestModalProps {
+  open: boolean;
+  accession: Accession2;
+  onClose: () => void;
+  selectedTest: ViabilityTest;
+  setNewViabilityTestOpened: React.Dispatch<React.SetStateAction<boolean>>;
+  setSelectedTest: React.Dispatch<React.SetStateAction<ViabilityTest | undefined>>;
+}
+
+export default function ViewViabilityTestModal(props: ViewViabilityTestModalProps): JSX.Element {
+  const { onClose, open, selectedTest, setNewViabilityTestOpened, setSelectedTest } = props;
+
+  const onCloseHandler = () => {
+    setSelectedTest(undefined);
+    onClose();
+  };
+
+  const openEdit = () => {
+    onClose();
+    setNewViabilityTestOpened(true);
```

Comment:
unrelated to this PR but I realized we need to do input validation for mandatory fields in the edit modal

Reply:
Yes, I was planning on adding validations at the end

Reply:
> Yes, I was planning on adding validations at the end

Sounds good 👍